README.md

```diff
@@ -288,6 +288,12 @@ const enum Enum { Two } // eslint-disable-line no-const-enum
 You can still disable rules with an .eslintrc.json, but should not in new packages.
 Disabling rules for the entire package makes it harder to review.
 
+##### Formatting with git hooks
```

Comment:
I just noticed where this was located in the repo; we have a section for formatting at https://github.com/DefinitelyTyped/DefinitelyTyped#are-files-formatted-automatically, perhaps this is better suited there?

(Sorry for the churn, I should have noticed where this was earier)

Reply:
No worries. I should have found that in the first place 😓